### PR TITLE
Remove the 'args' argument in service installation

### PR DESCRIPTION
**Pull request by TeoBotta90**
Drop the 'args' option since it doesn't work. Review the
syntax for sc command, Let the user configure the full binpath string in the "command" argument.

### CODE REVIEW

**Comment by wesQ3** [comment]
Hi, thanks for the patch.

We prefer descriptive commit messages in this project to make the project history easier to search in the future. Could you revise your commits to describe the changes in each with a short message (like the description in your pull request)?

**Comment by TeoBotta90** [comment]
> Hi, thanks for the patch.
> 
> We prefer descriptive commit messages in this project to make the project history easier to search in the future. Could you revise your commits to describe the changes in each with a short message (like the description in your pull request)?

Of couse @wesQ3, you see no description in the commits because I'm new to GitHub. I'll try to fix them asap

File: lib/Win32/ServiceManager.pm [hunk 1]
```diff
@@ -55,7 +55,7 @@ sub _nssm_install {
 }
 
 sub _sc_install {
-   qw(sc create), $_[1], qq(binpath= "$_[2]") . ($_[3] ?  " $_[3]" : ''),
+   qw(sc create), $_[1], qq(binpath= "$_[2]"),
```
Comment (TeoBotta90) [inline lib/Win32/ServiceManager.pm]: Drop the 'args' option since it doesn't work

File: lib/Win32/ServiceManager.pm [hunk 2]
```diff
@@ -115,6 +115,7 @@ sub create_service {
 
    my $name    = $args{name}    or die 'name is required!';
    my $display = $args{display} or die 'display is required!';
+   die "'args' argument not available" if $args{args};
```
Comment (TeoBotta90) [inline lib/Win32/ServiceManager.pm]: die in case of 'args' argument

File: lib/Win32/ServiceManager.pm [hunk 3]
```diff
@@ -141,10 +142,9 @@ sub create_service {
       if ($use_perl) {
          $command = $^X;
          die 'command is required!' unless $args{command};
-         $args = $args{command} . ($args{args} ? " $args{args}" : '')
```
Comment (TeoBotta90) [inline lib/Win32/ServiceManager.pm]: avoid appending the 'args' argument to the 'command' argument

File: lib/Win32/ServiceManager.pm [hunk 4]
```diff
@@ -141,10 +142,9 @@ sub create_service {
...
+         $args = $args{command}
       } else {
          $command = $args{command} or die 'command is required!';
-         $args = $args{args};
```
Comment (TeoBotta90) [inline lib/Win32/ServiceManager.pm]: removed $args value

**Comment by TeoBotta90** [comment]
> Hi, thanks for the patch.
> 
> We prefer descriptive commit messages in this project to make the project history easier to search in the future. Could you revise your commits to describe the changes in each with a short message (like the description in your pull request)?

Please be patient, I've put some comments on each piece of code that I've modified. Is it ok? Otherwise, please explain what should I do. Thanks!

**Comment by wesQ3** [comment]
I can see you're new. That's okay, everyone has to start somewhere and I'm glad you want to help :) 

Those are review comments which you've made on this pull request; I was referring to the commit messages on the commits themselves. Take a look at [the master branch on your fork](https://github.com/TeoBotta90/Win32-ServiceManager/commits/master) of the repo, see how the previous commits have messages describing what was changed? When you make a commit your git client will prompt you to add a message to describe the change you made.

Do you have a git client to revise your commits?

**Comment by TeoBotta90** [comment]
Hi, thanks for your help. I'll download a client that allows me to revise
my commits

Il giorno mar 19 nov 2019 alle ore 21:30 Wes Malone <
notifications@github.com> ha scritto:

**Comment by TeoBotta90** [comment]
Hi, I've downloaded this one https://www.gitkraken.com/git-client
I'd like to fix all the remaining issues in this fork, are you available
for a quick WebEx?
If so, please tell me when you're available and I'll arrange it.

Thanks, best regards

Il giorno mar 19 nov 2019 alle ore 21:30 Wes Malone <
notifications@github.com> ha scritto:

**Pull request closed without merging**